draft-ietf-ppm-dap.md

```diff
@@ -2599,83 +2502,69 @@ mitigations available to Aggregators also apply to the Leader.
 
 #### Capabilities
 
-1. Input validity proof verification. The Leader can forge proofs and collude
-   with a malicious Client to trick Aggregators into aggregating invalid inputs.
-     * This capability is no stronger than any Aggregator's ability to forge
-       validity proof in collusion with a malicious Client.
-1. Relaying messages between Aggregators. The Leader can compromise availability
-   by dropping messages.
-     * This capability is no stronger than any Aggregator's ability to refuse to
-       emit aggregate shares.
-1. Shrinking the anonymity set. The Leader instructs Aggregators to construct
-   output parts and so could request aggregations over few inputs.
-
-#### Mitigations
-
-1. Aggregators enforce agreed upon minimum aggregation thresholds to prevent
-   deanonymizing.
-1. If Aggregator output satisfies differential privacy {{dp}}, then genuine
-   records are protected regardless of the size of the anonymity set.
-
-### Collector
-
-#### Capabilities
-
-1. Advertising shared configuration parameters (e.g., minimum thresholds for
-   aggregations, joint randomness, arithmetic circuits).
-1. Collectors may trivially defeat availability by discarding aggregate shares
-   submitted by Aggregators.
-1. Known input injection. Collectors may collude with Clients to send known
-   input to the Aggregators, allowing Collectors to shrink the effective
-   anonymity set by subtracting the known inputs from the final output. Sybil
-   attacks {{Dou02}} could be used to amplify this capability.
-
-#### Mitigations
+1. Shrinking the anonymity set. The Leader instructs the Helper to construct
+   aggregate shares and so could request aggregations over dangerously few
+   reports.
+   1. This capability is particularly strong in the case of fixed-size queries
+      ({{fixed-size-query}}), because in that setting, the Leader is responsible
+      for assigning reports to batches and so can craft batches to target
+      certain contributions.
+   * This is mitigated by choosing a sufficient minimum batch size for the task.
+   * If aggregate shares emitted by Aggregators satisfy differential privacy
+     {{dp}}, then genuine records are protected regardless of the size of the
+     anonymity set.
+1. Relaying messages between Helper and Collector in the collect sub-protocol.
+   These messages are not authenticated, meaning the leader can:
+   1. Send collect parameters to the Helper that do not reflect the parameters
+      chosen by the Collector
```

Comment:
This is partly mitigated by the encryption of the aggregate share including the `BatchSelector` in its AAD. The aggregation parameter could still be rewritten and go undetected, but as noted below this isn't in the scope of the security model.

Reply:
Good catch! I also now wonder why the aggregate share isn't included in `AggregateShareAad`.